### ISSUE TITLE
[JET Connect] Flyt is no longer used

### ISSUE THREAD

**Issue by janaina-wittner** [HubRise/website#827]
Hi Antoine, Flyt is no longer used. I think we should update our site accordingly, and this update should include the URL, the Bridge and so on.

**Comment by antmonnier** [comment]
@janaina-wittner Yep ok, will do it because it needs to be done consistently across the bridge, documentation and wiki.
